Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -203,5 +205,7 @@
     sprite: Objects/Weapons/Melee/e_sword_double.rsi
   - type: Reflect
     enabled: true
-    reflectProb: .75
+    reflectProb: 1
```

Comment:
Will need to be discussed. Full reflection can be a potential issue.

Reply:
its just for laser weapons and the armory has more ballistic weapons than laser weapons, its fine

Reply:
^^